pym/bob/scm/url.py

```diff
@@ -40,11 +40,11 @@ class UrlScm(Scm):
     ]
 
     EXTRACTORS = {
-        "tar"  : ("tar xf", "--strip-components={}"),
-        "gzip" : ("gunzip -kf", None),
-        "xz"   : ("unxz -kf", None),
-        "7z"   : ("7z x -y", None),
-        "zip"  : ("unzip -o", None),
+        "tar"  : ("tar xf", "--strip-components={}", "--no-same-owner --no-same-permissions"),
```

Comment:
I think another field is not needed. Shouldn't `tar -x --no-same-owner --no-same-permissions -f` as command suffice?